src/classes/Trades.ts

```diff
@@ -912,6 +920,8 @@ export default class Trades {
                             Object.keys(dataDict[side])
                                 .map(sku => {
                                     if (!dataDict[side][sku] || getPureValue(sku as any) !== 0) return 0;
+                                    if (isWACEnabled && weapons.includes(sku)) return 0.05;
```

Comment:
We count in scraps so shouldn't this be (half a scrap) 0.5 * dataDict[side][sku] (which is the amount of the item)

Reply:
Oh right thank you.

Reply:
`dataDict[side][sku]` shouldn't be undefined.

Reply:
Are you talking about the line `922`?

Reply:
No no.

Reply:
One no was enough 😭.